index.js

```diff
@@ -6,43 +6,47 @@ const DEFAULT_APP_NAME = 'node';
 
 /**
  * @param {object} opts An object containing the required configuration.
- *    app                         - An express applcation
- *    appName                     - The name to be used as a default label
- *    ignoredRoutes (optional)    - An array of routes that should be ignored
+ *    appName                           - The name to be used as a default label
+ *    ignoredRoutes (optional)          - An array of routes that should be ignored
+ *    collectDefaultMetrics (optional)  - A boolean indicating whether or not to collect default metrics (default: true)
  * @returns {object} The internal prometheus client, middleware function, and metrics function.
  */
-module.exports = ({appName = DEFAULT_APP_NAME, ignoredRoutes = DEFAULT_IGNORED_ROUTES}) => {
+module.exports = ({appName = DEFAULT_APP_NAME, ignoredRoutes = DEFAULT_IGNORED_ROUTES, collectDefaultMetrics = false}) => {
 	Prometheus.register.setDefaultLabels({appName});
-	Prometheus.collectDefaultMetrics({timeout: DEFAULT_TIMEOUT_MS});
 
-	const durrationSummary = createDurrationSummary();
+	const durationSummary = createDurationSummary();
 	const statusCounter = createStatusCounter();
+    let defaultMetricsInterval = null;
 
-	/**
-   * This is the middleware function that actually measures response metrics. For every request the
-   * durration is measured as well as the request path, request method, and response status.
-   * @param {Request} req A request object
-   * @param {Response} res A responsee object
-   * @param {Function} next Function to triggre the next middleware function in the chain
-   */
+    if (collectDefaultMetrics) {
+        defaultMetricsInterval = Prometheus.collectDefaultMetrics({timeout: DEFAULT_TIMEOUT_MS});
```

Comment:
Maybe add timeout to options object as well? Or is that external?

Reply:
This timeout is how often the prometheus client probes the system to measure metrics, but the component that actually decides the resolution of the data is the one that pulls the metrics that are exposed. For our current setup, we pull metrics every ~1min.